cloud-common/src/main/java/com/sequenceiq/cloudbreak/common/type/ScalingHardLimits.java

```diff
@@ -0,0 +1,19 @@
+package com.sequenceiq.cloudbreak.common.type;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+/**
+ * Created by gpapp on 6/12/18.
+ */
+@Component
+public class ScalingHardLimits {
+    private static final int DEFAULT_UPSCALING_MAXIMUM_NODE_COUNT = 100;
```

Comment:
Static property not necessary

Reply:
True, but then stylecheck complained: it does not like variables with underscore.